### PR TITLE
Export comments to PDF.

### PR DESCRIPTION
**Pull request by jmcarp**
Proposed comment pipeline:
* JSON to markdown
* Markdown to HTML
* HTML to PDF

### CODE REVIEW

File: regulations/tasks.py [hunk 1]
```diff
@@ -57,10 +62,26 @@ def publish_metadata(response, key):
...
+        html_path = os.path.join(path, 'document.html')
+        pdf_path = os.path.join(path, 'document.pdf')
+        with open(html_path, 'w') as fp:
+            fp.write(html)
+        subprocess.check_output(['wkhtmltopdf', html_path, pdf_path])
+        pdf_file = open(pdf_path)
+        yield pdf_file
+    finally:
+        pdf_file.close()
+        shutil.rmtree(path)
```
Comment (jmcarp) [inline regulations/tasks.py]: :+1: 

Now that you mention it, there's a similar issue in `build_attachments`--if we die in the comprehension over files, we lose references to the file pointers and error on trying to close them.

File: regulations/management/commands/fetch_wkhtmltox.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    'wkhtmltox-0.12.3_linux-generic-amd64.tar.xz',
+)
+
+
+class Command(BaseCommand):
+    help = 'Fetch wkhtmltox binary'
+
+    def handle(self, **options):
+        subprocess.check_call(['wget', '-r', os.path.join(*BIN_PATH)])
+        subprocess.check_call(['tar', 'xzvf', BIN_PATH[-1]])
```
Comment (jmcarp) [inline regulations/management/commands/fetch_wkhtmltox.py]: This downloads a linux binary, so it's going to be hard to make it portable. I'm changing to error if the user's not running linux so we're explicit about this.

**Comment by jmcarp** [comment]
Now with unstyled pdf previews

5dzqbxcdvb

@cmc333333 do you want the wkhtmltox install script to live in n&c? Like I was saying, I'm not crazy about increasing the amount of stuff we have to copy and paste to make a new repo, but I also don't mind moving the script.

**Comment by cmc333333** [comment]
Nice work on the two step generation process. @jehlers you might want to see^

**Comment by jehlers** [comment]
Oooooo interesting! :smile: 

It took me a minute to figure this out, the user will have to "build" their comment before they can download it?

**Comment by jmcarp** [comment]
@jehlers right, it's a two-step process at the moment. Is that going to be confusing to users? It's possible to make it work with one click, but the workarounds I've thought of are less than elegant, technically.

**Comment by jehlers** [comment]
@jmcarp We can make it work if necessary as a two step process. I'll definitely want some language thoughts from @brittag! But IF we have time / find a good workaround, one step is definitely the ideal.

**Pull request merged**